components/scream/scripts/gather_all_data.py

```diff
@@ -3,7 +3,7 @@
 import os, pathlib
 import concurrent.futures as threading3
 from machines_specs import get_mach_env_setup_command, get_mach_batch_command, get_mach_testing_resources, \
-                           get_mach_cxx_compiler, get_mach_f90_compiler, get_mach_c_compiler
+                           get_mach_cxx_compiler, get_mach_f90_compiler, get_mach_c_compiler, is_cuda_machine
```

Comment:
Wondering why github is attributing these changes to your PR? Did something go wrong with the upstream merge?